### PR TITLE
Polyhedron demo : On-the-fly Selection_item

### PR DESCRIPTION
**Pull request by maxGimeno**
This PR acts on the Selection_plugin in the Polyhedron Demo. Currently, if the user has only one item with an associated selection_item, but several scene_items, if he selects on any item, the selection is only applied in the selection_item. With this PR, a selection_item is automatically created for the item being selected if none exists.

### CODE REVIEW

File: Polyhedron/demo/Polyhedron/Scene_polyhedron_item_k_ring_selection.h [hunk 1]
```diff
@@ -89,13 +89,29 @@ public Q_SLOTS:
...
+      poly_item->select(orig.x, orig.y, orig.z, dir.x, dir.y, dir.z);
+    }
+  }
 Q_SIGNALS:
   void selected(const std::set<Polyhedron::Vertex_handle>&);
   void selected(const std::set<Polyhedron::Facet_handle>&);
   void selected(const std::set<edge_descriptor>&);
   void toogle_insert(const bool);
   void endSelection();
+  void selection_request(QEvent *);
```
Comment (sloriot) [inline Polyhedron/demo/Polyhedron/Scene_polyhedron_item_k_ring_selection.h]: selectionRequest?

**Comment by sloriot** [comment]
TODO: @maxGimeno did you see my remark above?

**Comment by maxGimeno** [comment]
It's renamed.

**Comment by lrineau** [comment]
@maxGimeno The renaming is probably not yet pushed to your fork `mine`, because the old name is still visible in the changes.

**Comment by maxGimeno** [comment]
There.

**Comment by sloriot** [comment]
See #1099

**Pull request closed without merging**